### PR TITLE
refactor(share/ipld): Use consts for node type determination

### PR DESCRIPTION
**Pull request by renaynay**
Self-explanatory

### CODE REVIEW

File: share/ipld/namespace_hasher.go [hunk 1]
```diff
@@ -41,8 +40,7 @@ func (n *namespaceHasher) Write(data []byte) (int, error) {
 		return 0, fmt.Errorf("ipld: only one write to hasher is allowed")
 	}
 
-	ln, nln, hln := len(data), int(n.NamespaceLen), n.Hash.Size()
-	innerNodeSize, leafNodeSize := (nln*2+hln)*2, nln+appconsts.ShareSize
+	ln := len(data)
 	switch ln {
```
Comment (Wondertan) [inline share/ipld/namespace_hasher.go:44]:
```suggestion
	switch len(data) {
```

Comment (renaynay) [reply share/ipld/namespace_hasher.go:44]: we return ln in error so no

Comment (Wondertan) [reply share/ipld/namespace_hasher.go:44]: Then we might also want to use it on line 56

Comment (renaynay) [reply share/ipld/namespace_hasher.go:44]: done

**Review by Wondertan — APPROVED** [review]

**Review by Bidon15 — APPROVED** [review]
🚢

**Review by walldiss — APPROVED** [review]

**Review by MSevey — APPROVED** [review]

**Pull request merged**